apis/printservice/printservice.yaml

```diff
@@ -0,0 +1,211 @@
+            responses:
+                '200':
+                    description: 'OK ! Operation successful."Certificate addition" operation was executed successfully.'
+                    schema:
+                        allOf:
+                            - $ref: 'https://raw.githubusercontent.com/project-sunbird/project-sunbird.github.io/dev/apis/definitions/envelope.yaml#/ApiResponse'
+                            - $ref: '#/definitions/PrintPdfResponse'
+                '400':
+                    description: 'BAD REQUEST. The "Certificate addition" operation failed. Input for a mandatory parameter may be missing.'
+                    schema:
+                        $ref: '#/definitions/PdfGenErrorResponse'
+                '500':
+                    description: 'INTERNAL SERVER ERROR ! Looks like something went wrong. These errors are tracked automatically. Try refreshing the page. If the problem persists, contact info@sunbird.org. '
+
+
+definitions:
+
+    PdfGenerationRequest:
+        title: Certificate Verify request
+        type: object
+        properties:
+            request:
+                $ref: '#/definitions/genPdfRequest'
+        required:
+            - request
+
+    genPdfRequest:
+        type: object
+        description: htmlTemplate zip url should be provided always while making requests.
+        properties:
+            htmlTemplate:
+                description: The htmlTemplate zip url should contain index.html(with placeholders) and its related css & images file, which are required to generate certificate.
+                type: string
+            context:
+                description: This will contain the placeHolder and it's value which needs to be replaced in HTML.
+                type: object
+                example:
+                    recipientName: John Deo
+                    certificateName: ABC Certificate
+            storageParams:
+                type: object
+                description: this object will have the container name and path where its needs to be stored in cloud. it not provided it will stored at default location.
+                properties:
+                    path: {
+                        description: path inside container,
+                        type: string,
+                    }
+                    containerName: {
+                        description: name of container,
+                        type: string
+                    }
+                example:
+                    path: pathToFoler/
+                    containerName: nameOfContainer
+        required:
+            - htmlTemplate
+
+    PrintPdfResponse:
+        description: PDF response
+        type: object
+        properties:
+            result:
+                description: The response details
+                type: object
+                properties:
+                    pdfUrl:
+                        description: The downloadable url of the pdf, public in nature
```

Comment:
@anmol2302 modify the sentence to, 'The public URL from where the pdf can be downloaded.'